ci-operator/config/openshift/kube-state-metrics/master.json

```diff
@@ -0,0 +1,35 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    }
+  },
+  "canonical_go_repository": "github.com/openshift/kube-state-metrics",
```

Comment:
Since this repo is a fork, you should set this value to the origin `k8s.io/kube-state-metrics` (whatever the go import line would be for the repo).  Set this for the others the same.

Reply:
Good point. Updated for `prometheus-operator`,  `prometheus-config-reloader` and `kube-state-metrics`.